### PR TITLE
Minor refactoring to expose functions to upstream libraries

### PR DESCRIPTION
**Pull request by live-wire**

### CODE REVIEW

File: pkg/flink/handler.go [hunk 1]
```diff
@@ -151,7 +158,7 @@ func (h flinkResourceHandler) OnAbort(ctx context.Context, tCtx pluginsCore.Task
 	return abortBehavior, nil
 }
 
-func flinkClusterTaskLogs(ctx context.Context, flinkCluster *flinkOp.FlinkCluster) ([]*core.TaskLog, error) {
+func FlinkClusterTaskLogs(ctx context.Context, flinkClusterName string, flinkClusterNamespace string) ([]*core.TaskLog, error) {
```
Comment (regadas) [inline pkg/flink/handler.go:161]: I think it would still be better to pass in the FlinkCluster here.

**Review by regadas — APPROVED** [review]
Minor comment otherwise LGTM.

File: pkg/flink/handler.go (hunk 1 above)
Comment (live-wire) [reply pkg/flink/handler.go:161]: Currently trying to depend on this library for another flyte webApi plugin.
For some flyte plugins, we might not even have a `FlinkCluster` object, but only metadata about a FlinkCluster.

Comment (regadas) [reply pkg/flink/handler.go:161]: Maybe it's better to come with a type here to encapsulate more metadata in the future if needed

Comment (live-wire) [reply pkg/flink/handler.go:161]: Yes. Makes sense. 
Will do it next time we add any more information to taskLogs. 👍

**Pull request merged**